Modules/ControlPanel/ControlPanel.lua

```diff
@@ -9,6 +9,7 @@ local GUTIL = CraftSim.GUTIL
 ---@class CraftSim.CONTROL_PANEL.FRAME : GGUI.Frame
 CraftSim.CONTROL_PANEL.frame = nil
 
+local systemPrint = print;

```

Comment:
please remove if not used or use `CraftSim.DEBUG:SystemPrint`

Reply:
It's used line 201 https://github.com/derfloh205/CraftSim/blob/a290db1078678134f3f6da365f8a373376b77645/Modules/ControlPanel/ControlPanel.lua#L201